### ISSUE TITLE
what's the default checkpoint monitor in 0.10.0?

### ISSUE THREAD

**Issue by xiadingZ** [PyTorchLightning/pytorch-lightning#3984]
what's the default checkpoint monitor in 0.10.0? `loss` or `val_loss` returned in `validation_step`?

**Comment by SkafteNicki** [comment]
The default checkpoint callback is initialized here:
https://github.com/PyTorchLightning/pytorch-lightning/blob/53e056612c1d27deb45916351222522efabfaf35/pytorch_lightning/trainer/connectors/callback_connector.py#L48
Since `monitor=None` as default, this means that a checkpoint will be saved after each epoch (with only the last one being save as default). So no value is monitored.
